src/journal/journald-context.h

```diff
@@ -55,6 +56,13 @@ struct ClientContext {
 
         usec_t log_ratelimit_interval;
         unsigned log_ratelimit_burst;
+
+#if HAVE_PCRE2
+        char *log_include_regex;
+        char *log_exclude_regex;
```

Comment:
I would put the minimal amount of code inside `HAVE_PCRE2`, so I'd still read the cgroup xattr etc even if we're built with out PCRE2, just skip compiling and actually using the regexes.